protocol/directory.go

```diff
@@ -39,18 +65,43 @@ func (d *ConiksDirectory) Update() {
 	}
 }
 
+// SetPolicies sets this ConiksDirectory's epoch deadline and VRF
+// private key.
 func (d *ConiksDirectory) SetPolicies(epDeadline merkletree.TimeStamp, vrfKey vrf.PrivateKey) {
 	d.policies = merkletree.NewPolicies(epDeadline, vrfKey)
 }
 
+// EpochDeadline returns this ConiksDirectory's current epoch deadline
+// as a timestamp.
 func (d *ConiksDirectory) EpochDeadline() merkletree.TimeStamp {
 	return d.pad.LatestSTR().Policies.EpochDeadline
 }
 
+// LatestSTR Returns this ConiksDirectory's latest STR.
```

Comment:
But lookups in the "current" epoch if "current" is `t+1` search the tree for `t` *because* the tree root hasn't been signed yet. So why doesn't it make sense to say that `t` is the current epoch if `t+1` doesn't exist and hasn't been signed yet?